src/workers/cdc/primary.ts

```diff
@@ -663,6 +665,17 @@ const onSourceChange = async (
     if (data.payload.before!.private !== data.payload.after!.private) {
       await notifySourcePrivacyUpdated(logger, data.payload.after!);
     }
+
+    const beforeFlags = data.payload.before.flags as unknown as string;
+    const afterFlags = data.payload.after.flags as unknown as string;
```

Comment:
There is an issue with the type from CDC, tried fixing it but it is giving more errors than it fixed. Let's probably tackle it some other time.

Reply:
yes, its ok to tackle separate, tnx for trying